deployments/constructs/platforms/azure-platform.ts

```diff
@@ -0,0 +1,176 @@
+/*
+ * Copyright (c) 2023 - for information on the respective copyright owner
```

Comment:
update copyright header. New file -> 2024, updated / moved -> 2023 - 2024

Reply:
updated all copyright headers to 2024